### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by MaiRizk**
#### Here're the steps completed in this project;

- Create a repo and a feature branch (hello-microverse).
- Add .gitignote folder to contain node_modules and test.md file.
- Setup linters for HTML and CSS.
- Update Readme file to describe the project.
- Create index.html file with h1 header.
- Create styles.css file to style the header.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Hello Mictoverse</title>
+  </head>
+  <body>
+    <h1>Hello Microverse!</h1>
```
Comment (OmarRamoun) [inline index.html:11]: - [ ] [REQUIRED] Please, make sure to add a class to this `h1` tag as requested in project requirements 🧾.

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @MaiRizk 👋

## Welcome to Microverse! 🌎

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide on how to fix them down below. 🤓_

## Highlights 🌟

- ✔️ descriptive commit messages
- ✔️ descriptive PR title and description
- ✔️ no linter errors

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better 👍. So, you can skip them if you don't have the time but remember to come back and implement them because they are going to enhance the quality of your code 💯_

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@ramounramoun](https://twitter.com/ramounramou)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MaiRizk) [reply index.html:11]: @OmarRamoun Thanks for the review. 
I have rechecked the requirements and nothing mentions I need to your classes. The only requirement for the HTML file is to use h1.
Can you please recheck?

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @MaiRizk,  This is @sja-thedude your code reviewer.

Congratulations! 🎉 🎉 🎉 🎉  Your first project is complete 😄 !  There is nothing else to say other than... it's time to merge it :shipit: 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**